dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,10 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        import dstep.core.Traits : isKeyword;
+        if (ident.spelling.isKeyword())
+            return false;
```

Comment:
I see, there seems to be a bit of a clash between your example and the one I have. To be more, specific, issue I am trying to solve is translation of this macro:

```c
#define PROTOBUF_C_ASSERT(condition) assert(condition)
```

Here `assert` resolves to function in `assert.h` in C header, but exiting translation will try turning it into `alias PROTOBUF_C_ASSERT = asser` which of course can't work.